internal/pkg/cli/dialog/use_template.go

```diff
@@ -141,7 +141,14 @@ func (d *useTmplDialog) askInput(inputDef input.Input) error {
 			Default: cast.ToString(inputDef.Default),
 			Hidden:  inputDef.Kind == input.KindHidden,
 		}
-		value, _ := d.Ask(question)
+
+		var value string
+		if inputDef.Kind == input.KindTextarea {
+			value, _ = d.Editor("txt", question)
+		} else {
+			value, _ = d.Ask(question)
+		}
```

Comment:
Pri dokoncovani dokumentacie som si vsimol, ze `textarea` sa zadava ako `input`, tj, nie je mozne zadat viacero riadkov.
Preto som to upravil, aby sa hodnota zadavala cez edtor.